server-phoenix/test/helios_web/controllers/web_hooks/slack_controller_test.exs

```diff
@@ -0,0 +1,113 @@
+defmodule HeliosWeb.WebHooks.SlackControllerTest do
+  use HeliosWeb.ConnCase, async: true
+  use ExUnit.Case
+  alias Helios.Event
+  alias Helios.Repo
+
+  test "connection response OK", %{conn: conn} do
+    conn =
+      conn
+      |> put_req_header("content-type", "application/json")
+      |> post(
+        Routes.slack_path(conn, :handle),
+        %{
+          token: "Sim-123456",
+          team_id: "TXXXXXXXX",
+          api_app_id: "AXXXXXXXX",
+          event: %{
+            client_msg_id: "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
+            type: "message",
+            text: "hi",
+            user: "Priscila",
+            ts: "155984192.001600",
+            channel: "CXXXXXXXX",
+            event_ts: "1559841925.001600",
+            channel_type: "channel"
+          },
+          type: "event_callback",
+          event_id: "<%= @event_id %>",
+          event_time: 1_559_841_923,
+          authed_users: ["UXXXXXXXX", "UXXXXXXXX"],
+          slack: %{
+            token: "Sim-123456",
+            team_id: "TXXXXXXXX",
+            api_app_id: "AXXXXXX",
+            event: %{
+              client_msg_id: "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
+              type: "message",
+              text: "hi",
+              user: "UXXXXXXXX",
```

Comment:
Question for follow up work. Do we get a user identifier or a name in the above `event` map?